submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/converters/to/FileListDocFileConverter.kt

```diff
@@ -0,0 +1,18 @@
+package ac.uk.ebi.biostd.persistence.doc.db.converters.to
+
+import ac.uk.ebi.biostd.persistence.doc.db.converters.shared.FileListDocFileFields
+import ac.uk.ebi.biostd.persistence.doc.model.FileListDocFile
+import org.bson.Document
+
+class FileListDocFileConverter {
+    fun convert(fileListDocFile: FileListDocFile): Document {
+        val fileListDocFileDoc = Document()
+        fileListDocFileDoc[CommonsConverter.classField] = FileListDocFileFields.FILE_LIST_DOC_FILE_CLASS
```

Comment:
You could add full imports to make the code less verbose i.e. `FILE_LIST_DOC_FILE_CLASS` instead of `FileListDocFileFields.FILE_LIST_DOC_FILE_CLASS`